lua/gatekeeper_config.lua

```diff
@@ -20,6 +20,9 @@ function gatekeeper_init()
 	local llsf = require("lls")
 	local lls_conf = llsf(net_conf, numa_table)
 
+	local cpsf = require("cps")
+	local cps_conf = cpsf(net_conf, numa_table)
+
```

Comment:
It's better to move the initialization of the CPS block after the initialization of the GK/GT/GK-GT-unit blocks in order to increase the chance that the LLS block is allocated in the same NUMA node as the GK/GT/GK-GT-unit blocks. A comment explaining this is a good idea since it's not an obvious idea.